code/modules/emotes/custom_emote.dm

```diff
@@ -68,6 +68,10 @@
 		var/list/m_viewers = in_range["mobs"]
 		var/list/o_viewers = in_range["objs"]
 
+		for(var/obj/o in contents)
+			o_viewers |= o
+
```

Comment:
If anyone's wondering, this is because w/ current behavior worn items aren't count as "in view". I figure this is sane enough to be the solution to that, as you'd assume that your shoes can see an emote you do or whatever.